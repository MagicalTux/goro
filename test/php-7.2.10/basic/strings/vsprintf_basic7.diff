     *** Testing vsprintf() : basic functionality - using unsigned format ***
002- string(10) "4294966185"
003- string(21) "4294966185 4293732729"
004- string(32) "4294966185 4293732729 4292621864"
002+ string(22) "1777777777777777775651"
003+ string(45) "1777777777777777775651 1777777777777773224571"
004+ string(68) "1777777777777777775651 1777777777777773224571 1777777777777767033050"
     Done
