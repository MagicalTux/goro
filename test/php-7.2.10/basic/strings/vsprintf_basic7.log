
---- EXPECTED OUTPUT
*** Testing vsprintf() : basic functionality - using unsigned format ***
string(10) "4294966185"
string(21) "4294966185 4293732729"
string(32) "4294966185 4293732729 4292621864"
Done
---- ACTUAL OUTPUT
*** Testing vsprintf() : basic functionality - using unsigned format ***
string(22) "1777777777777777775651"
string(45) "1777777777777777775651 1777777777777773224571"
string(68) "1777777777777777775651 1777777777777773224571 1777777777777767033050"
Done
---- FAILED
