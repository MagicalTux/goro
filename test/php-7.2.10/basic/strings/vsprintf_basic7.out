*** Testing vsprintf() : basic functionality - using unsigned format ***
string(22) "1777777777777777775651"
string(45) "1777777777777777775651 1777777777777773224571"
string(68) "1777777777777777775651 1777777777777773224571 1777777777777767033050"
Done